constants.py

```diff
@@ -0,0 +1,4 @@
+VERSION = "1.3.4"
+PKGNAME = "panoply-python-sdk"
+MAXSIZE = 1024 * 250  # 250kib
+FLUSH_TIMEOUT = 2.0   # 2 seconds
```

Comment:
the last two shouldn't be outside of panoply folder. please consider moving them to a specific file.